gapis/api/vulkan/api/vk11structs.api

```diff
@@ -110,148 +126,194 @@ class VkDescriptorUpdateTemplateEntry {
   size             stride
 }
 
+@structextends("VkBindSparseInfo")
 class VkDeviceGroupBindSparseInfo {
+  @values(VK_STRUCTURE_TYPE_DEVICE_GROUP_BIND_SPARSE_INFO)
   VkStructureType sType
   const void*     pNext
   u32             resourceDeviceIndex
   u32             memoryDeviceIndex
 }
 
+@structextends("VkCommandBufferBeginInfo")
 class VkDeviceGroupCommandBufferBeginInfo {
+  @values(VK_STRUCTURE_TYPE_DEVICE_GROUP_COMMAND_BUFFER_BEGIN_INFO)
   VkStructureType sType
   const void*     pNext
   u32             deviceMask
 }
 
+@structextends("VkDeviceCreateInfo")
 class VkDeviceGroupDeviceCreateInfo {
+  @values(VK_STRUCTURE_TYPE_DEVICE_GROUP_DEVICE_CREATE_INFO)
   VkStructureType         sType
   const void*             pNext
   u32                     physicalDeviceCount
+  @len("physicalDeviceCount")
   const VkPhysicalDevice* pPhysicalDevices
 }
 
+@structextends("VkRenderPassBeginInfo")
 class VkDeviceGroupRenderPassBeginInfo {
+  @values(VK_STRUCTURE_TYPE_DEVICE_GROUP_RENDER_PASS_BEGIN_INFO)
   VkStructureType sType
   const void*     pNext
   u32             deviceMask
   u32             deviceRenderAreaCount
+  @len("deviceRenderAreaCount")
   const VkRect2D* pDeviceRenderAreas
 }
 
+@structextends("VkSubmitInfo")
 class VkDeviceGroupSubmitInfo {
+  @values(VK_STRUCTURE_TYPE_DEVICE_GROUP_SUBMIT_INFO)
   VkStructureType sType
   const void*     pNext
   u32             waitSemaphoreCount
+  @len("waitSemaphoreCount")
   const u32*      pWaitSemaphoreDeviceIndices
   u32             commandBufferCount
+  @len("commandBufferCount")
   const u32*      pCommandBufferDeviceMasks
   u32             signalSemaphoreCount
+  @len("signalSemaphoreCount")
   const u32*      pSignalSemaphoreDeviceIndices
 }
 
 class VkDeviceQueueInfo2 {
+  @values(VK_STRUCTURE_TYPE_DEVICE_QUEUE_INFO_2)
   VkStructureType          sType
   const void*              pNext
   VkDeviceQueueCreateFlags flags
   u32                      queueFamilyIndex
   u32                      queueIndex
 }
 
+@structextends("VkFenceCreateInfo")
 class VkExportFenceCreateInfo {
+  @values(VK_STRUCTURE_TYPE_EXPORT_FENCE_CREATE_INFO)
   VkStructureType                sType
   const void*                    pNext
   VkExternalFenceHandleTypeFlags handleTypes
 }
 
+@structextends("VkMemoryAllocateInfo")
 class VkExportMemoryAllocateInfo {
+  @values(VK_STRUCTURE_TYPE_EXPORT_MEMORY_ALLOCATE_INFO)
   VkStructureType                 sType
   const void*                     pNext
   VkExternalMemoryHandleTypeFlags handleTypes
 }
 
+@structextends("VkSemaphoreCreateInfo")
 class VkExportSemaphoreCreateInfo {
+  @values(VK_STRUCTURE_TYPE_EXPORT_SEMAPHORE_CREATE_INFO)
   VkStructureType                    sType
   const void*                        pNext
   VkExternalSemaphoreHandleTypeFlags handleTypes
 }
 
+@returnedonly
```

Comment:
Anything with pNext and an sType cannot be returnedonly.

Reply:
Changed to returnedchain